src/main/storm/mesos/MesosNimbus.java

```diff
@@ -789,15 +789,11 @@ protected void computeResourcesForSlot(final RotatingMap<OfferID, Offer> offers,
         executorInfoBuilder
             .setName(executorName)
             .setExecutorId(ExecutorID.newBuilder().setValue(details.getId()))
-            .setData(ByteString.copyFromUtf8(executorDataStr));
-        if (!subtractedExecutorResources) {
-          subtractedExecutorResources = true;
```

Comment:
I think we need to add this assignment back in. With this change it no longer gets set to true in the rest of the code, and if `subtractedExecutorResources` means the same as it did before PR #65, this will mean that we're over-provisioning resources and accounting for the executor's resources in every task, instead of just in the first one.

@brndnmtthws Am I correct in assuming `subtractedExecutorResources` does essentially the same thing after the PR #65?